clang/test/CodeGen/zos-pragmas.c

```diff
@@ -0,0 +1,12 @@
+// REQUIRES: systemz-registered-target
+// RUN: %clang_cc1 -emit-llvm -triple s390x-none-zos -fvisibility=hidden %s -o - | FileCheck %s
+// expected-no-diagnostics
```

Comment:
There's no `-verify` line, so this comment doesn't do anything and can be removed.